src/java.base/share/classes/java/io/FileInputStream.java

```diff
@@ -165,9 +162,8 @@ public FileInputStream(File file) throws FileNotFoundException {
     }
 
     /**
-     * Creates a {@code FileInputStream} by using the file descriptor
-     * {@code fdObj}, which represents an existing connection to an
-     * actual file in the file system.
+     * Creates a {@code FileInputStream} to read from an existing file
+     * represented by the {@code FileDescriptor} object {@code fdObj}.
```

Comment:
I don't know if you meant to touch this constructor or not but I think it's okay to use "connected" here as this is creating a FIS to read bytes from whatever the file descriptor is connected to.

Reply:
I have reverted it locally and that will be in the next commit.